apiserver/backupoptions/backupoptionsutil.go

```diff
@@ -163,6 +163,17 @@ func isValidCompressLevel(compressLevel int) bool {
 	}
 }
 
+// isValidRetentionRange validates that pgBackrest Full, Diff or Archive
+// retention option value is set within the allowable range.
+// allowed: 1-9999999
+func isValidRetentionRange(retentionRange int) bool {
+	if retentionRange >= 1 && retentionRange <= 9999999 {
+		return true
+	} else {
+		return false
+	}
+}
```

Comment:
Could this be condensed to `return (retentionRange >= 1 && retentionRange <= 9999999)`?

Reply:
Sure, I don't see why not. I'll make the change.

Reply:
Updated, let me know if there's anything else that could be improved. Thanks!

Reply:
LGTM, I have no other suggestions that would be relevant to the scope of this PR. :+1: